edivorce/apps/core/utils/conditional_logic.py

```diff
@@ -123,9 +123,13 @@ def determine_missing_extraordinary_expenses(questions_dict):
 
     if questions_dict.get('special_extraordinary_expenses') == 'YES':
         for special_expense in special_expenses_keys:
-            value = questions_dict.get(special_expense)
-            if value and value != '0.00':
-                return False
+            value = questions_dict.get(special_expense, 0)
+            try:
+                as_num = float(value)
+                if as_num > 0:
+                    return False
+            except ValueError:
+                pass
```

Comment:
Do we want to return True if value is not a float?

Reply:
I don't think so. We are looking for expenses that are > $0, so the only time we should stop looking through the fields is when it returns a number > 0. It may be possible that the value ends up being an empty string, in which case, we want to skip it and keep searching.